recOrder/plugin/widget/main_widget.py

```diff
@@ -1996,31 +2005,37 @@ def enter_config_group(self):
         -------
 
         """
+        #if/else takes care of the clearing of config
+        if self.ui.cb_config_group.count()!=0:
+            self.mmc = self.bridge.get_core()
+            self.mm = self.bridge.get_studio()
 
-        # Gather config groups and their children
-        self.config_group = self.ui.cb_config_group.currentText()
-        config = self.mmc.getAvailableConfigs(self.config_group)
+            # Gather config groups and their children
+            self.config_group = self.ui.cb_config_group.currentText()
+            config = self.mmc.getAvailableConfigs(self.config_group)
 
-        channels = []
-        for i in range(config.size()):
-            channels.append(config.get(i))
+            channels = [] 
+            for i in range(config.size()):
+                channels.append(config.get(i))
 
-        # Check to see if any states are missing
-        states = ['State0', 'State1', 'State2', 'State3', 'State4']
-        missing = []
-        for state in states:
-            if state not in channels:
-                missing.append(state)
+            # Check to see if any states are missing
+            states = ['State0', 'State1', 'State2', 'State3', 'State4']
+            missing = []
+            for state in states:
+                if state not in channels:
+                    missing.append(state)
 
-        # if states are missing, set the combo box red and alert the user
-        if len(missing) != 0:
-            msg = f'The chosen config group ({self.config_group}) is missing states: {missing}. '\
-                   'Please refer to the recOrder wiki on how to set up the config properly.'
+            # if states are missing, set the combo box red and alert the user
+            if len(missing) != 0:
+                msg = f'The chosen config group ({self.config_group}) is missing states: {missing}. '\
+                    'Please refer to the recOrder wiki on how to set up the config properly.'
 
-            self.ui.cb_config_group.setStyleSheet("border: 1px solid rgb(200,0,0);")
-            raise KeyError(msg)
+                self.ui.cb_config_group.setStyleSheet("border: 1px solid rgb(200,0,0);")
+                raise KeyError(msg)
+            else:
+                self.ui.cb_config_group.setStyleSheet("")
         else:
-            self.ui.cb_config_group.setStyleSheet("")
+            print("CONFIG GROUP IS NOT EMPTY")
```

Comment:
Refer to the use of [this method from napari](https://github.com/mehta-lab/recOrder/blob/14faa3d33a02e0aba649cedc9b2f20e498d0a3f7/recOrder/plugin/workers/acquisition_workers.py#L13) to show a warning in the GUI.
It's also worth thinking about the intention of this message, so maybe it can be articulated in a way that guides the user to recover from the issue. If this can not be recovered, consider raising an error.
Some context: [#178](https://github.com/mehta-lab/recOrder/pull/178)